roles/openshift_metering/tasks/uninstall.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
should we also be removing the CRD as part of removal/clean up?

Reply:
Good question. I think that's a bit more dangerous as there may be multiple installs of it and deleting the CRD would cause a cascading deletion of all instances. I think for now it's probably okay to leave around, and I believe this is what's happening in other playbooks.